libs/expo/betterangels/src/lib/static/constants.ts

```diff
@@ -0,0 +1,197 @@
+import { ServiceEnum } from '../apollo';
+import { enumDisplayServices } from './enumDisplayMapping';
+
+export const Services = [
+  {
+    title: 'Immediate Needs',
+    items: [
+      { title: enumDisplayServices[ServiceEnum.Food], enum: ServiceEnum.Food },
+      {
+        title: enumDisplayServices[ServiceEnum.Water],
+        enum: ServiceEnum.Water,
+      },
+      {
+        title: enumDisplayServices[ServiceEnum.Blanket],
+        enum: ServiceEnum.Blanket,
+      },
+      { title: enumDisplayServices[ServiceEnum.Book], enum: ServiceEnum.Book },
+      {
+        title: enumDisplayServices[ServiceEnum.Clothes],
+        enum: ServiceEnum.Clothes,
+      },
+      {
+        title: enumDisplayServices[ServiceEnum.HygieneKit],
+        enum: ServiceEnum.HygieneKit,
+      },
+      {
+        title: enumDisplayServices[ServiceEnum.PetFood],
+        enum: ServiceEnum.PetFood,
+      },
+      {
+        title: enumDisplayServices[ServiceEnum.Shoes],
+        enum: ServiceEnum.Shoes,
+      },
+      {
+        title: enumDisplayServices[ServiceEnum.Shower],
+        enum: ServiceEnum.Shower,
+      },
+      { title: enumDisplayServices[ServiceEnum.Tent], enum: ServiceEnum.Tent },
+    ],
+  },
+  {
+    title: 'Communication',
+    items: [
+      {
+        title: enumDisplayServices[ServiceEnum.MailPickUp],
+        enum: ServiceEnum.MailPickUp,
+      },
+
+      {
+        title: enumDisplayServices[ServiceEnum.CaliforniaLifelinePhone],
+        enum: ServiceEnum.CaliforniaLifelinePhone,
+      },
+
+      {
+        title: enumDisplayServices[ServiceEnum.InternetAccess],
+        enum: ServiceEnum.InternetAccess,
+      },
+    ],
+  },
+  {
+    title: 'Identity',
+    items: [
+      {
+        title: enumDisplayServices[ServiceEnum.BirthCertificate],
+        enum: ServiceEnum.BirthCertificate,
+      },
+      {
+        title: enumDisplayServices[ServiceEnum.DmvNoFeeIdForm],
+        enum: ServiceEnum.DmvNoFeeIdForm,
+      },
+      {
+        title: enumDisplayServices[ServiceEnum.SocialSecurityCardReplacement],
```

Comment:
on my phone this one doesn't fit, so may need to wrap it

Reply:
fixed